bibtex/bst/biblatex/biblatex.bst

```diff
@@ -111,6 +111,10 @@ ENTRY {
     pubstate
     reprinttitle
     holder
+    fileddate
```

Comment:
Some more work is needed to properly handle `date` fields in the `.bst`. Cf. everything that involves `urldate` and its components `urlyear` etc.

Reply:
ok done